src/main/webapp/app/admin/system-notification-management/system-notification-management-detail.component.ts

```diff
@@ -6,9 +6,9 @@ import { SystemNotificationService } from 'app/core/system-notification/system-n
 
 @Component({
     selector: 'jhi-notification-mgmt-detail',
-    templateUrl: './notification-management-detail.component.html',
+    templateUrl: './system-notification-management-detail.component.html',
 })
-export class NotificationMgmtDetailComponent implements OnInit, OnDestroy {
+export class SystemNotificationMgmtDetailComponent implements OnInit, OnDestroy {
```

Comment:
Should we switch `Mgmt` to `Management`? I think the overall codebase tends to favor verbosity over abbreviations.

Reply:
I will change that :-)